ros2bag/ros2bag/api/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
Could this file just be empty and the dependency on rclpy removed from the package.xml ?